```
/home/runner/work/goldendict-ng/goldendict-ng/src/keyboardstate.cc:7:10: fatal error: QtVersionChecks: No such file or directory
    7 | #include <QtVersionChecks>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -69,7 +69,7 @@ if (WITH_TTS)
     list(APPEND GD_QT_COMPONENTS TextToSpeech)
 endif ()
 
-find_package(Qt6 REQUIRED COMPONENTS ${GD_QT_COMPONENTS})
+find_package(Qt6 6.2 REQUIRED COMPONENTS ${GD_QT_COMPONENTS})
 
 qt_standard_project_setup()
 set(CMAKE_AUTORCC ON) # not included in the qt_standard_project_setup

--- a/src/keyboardstate.cc
+++ b/src/keyboardstate.cc
@@ -8,8 +8,8 @@
 bool KeyboardState::checkModifiersPressed( int mask )
 {
   auto modifiers = QApplication::queryKeyboardModifiers();
-
-  return !( ( mask & Alt && !( modifiers.testFlag( Qt::AltModifier ) ) )
-            || ( mask & Ctrl && !( modifiers.testFlag( Qt::ControlModifier ) ) )
-            || ( mask & Shift && !( modifiers.testFlag( Qt::ShiftModifier ) ) ) );
+  return modifiers.testFlags( { ( mask & Alt ? Qt::AltModifier : Qt::NoModifier )
+                                | ( mask & Win ? Qt::MetaModifier : Qt::NoModifier )
+                                | ( mask & Ctrl ? Qt::ControlModifier : Qt::NoModifier )
+                                | ( mask & Shift ? Qt::ShiftModifier : Qt::NoModifier ) } );
 }

